packages/ui-react/tsconfig.json

```diff
@@ -19,7 +19,7 @@
         "name": "typescript-plugin-css-modules",
         "options": {
           "rendererOptions": {
-            "includePaths": ["./styles"]
+            "includePaths": ["./styles", "./node_modules", "../../node_modules"]
```

Comment:
I'm curious to know what this fixes. Can you give an example?

Reply:
Maybe @ChristiaanScheermeijer can explain? I suspect it is so that the `@jwp/` imports are recognised by the ts plugin (which by the way still doesn't work).

Reply:
Yes, this tells the Sass compiler where to look for imports. Because this is a monorepo, we need to instruct the Sass compiler to look in both node_modules folders.

Reply:
Thanks!